_data/glossary/job.yaml

```diff
@@ -0,0 +1,8 @@
+id: job
+name: Job
```

Comment:
It would be good to add a `full-link` to https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/.

Reply:
Done. PTAL?

Reply:
I think it just needs to be `/docs/concepts/workloads/controllers/jobs-run-to-completion/`